UNITTESTS/target_h/platform/mbed_power_mgmt.h

```diff
@@ -0,0 +1,4 @@
+#ifndef MBED_POWER_MGMT_H
```

Comment:
Is there any reason to hide the function definition of the original file ? 
`platform` is supposed to be platform independent.